### PR TITLE
build windows git initializer

### PR DESCRIPTION
**Pull request by krancour**
First of at least two PRs collectively aimed at addressing #1713 

A follow-up will make chart and code changes so that the windows variant of this image is applied as necessary.

This PR is only focused on getting the image to build.

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ and well-documented pattern colloquially called "Docker in Docker" or simply
 
 Building a Windows-based OCI image inside a Windows-based OCI container is
 _not_, as of this writing, a common, well-understood, or well-documented
-pattern. Building Brigade's one Windows-based OCI image, therefore, requires the
-use of a physical Windows machine or virtual machine having an appropriate
+pattern. Building Brigade's two Windows-based OCI images, therefore, requires
```
Comment (vdice) [inline .github/workflows/README.md:15]: Wonder if we want to just simplify to 'Building Brigade's Windows-based OCI images' (here and elsewhere)... unless we're pretty certain it'll now be capped at the two?

**Review by vdice — APPROVED** [review]
LGTM

File: .github/workflows/README.md (hunk 1 above)
Comment (krancour) [reply .github/workflows/README.md:15]: Pretty sure it's limited to these two, but I like your suggestion regardless. Changing.

Comment (krancour) [reply .github/workflows/README.md]: All set.

**Pull request merged**